lib/string.js

```diff
@@ -70,6 +70,10 @@ internals.String.prototype._base = function (value, state, options) {
         if (this._flags.trim) {
             value = value.trim();
         }
+
+        if (this._flags.transform) {
```

Comment:
@Marsup I've thought about it, but then looking further up in `any.js` (lines 490 down) I can see a lot of edge cases that *MIGHT* come up when dealing with `default(...)` and `raw(...)`.

Definitely more than I can chew at the moment, I'd need to get way more acquainted with the code before tackling something of this magnitude :-)